modules/ivc_champva/app/models/ivc_champva/vha_10_10d.rb

```diff
@@ -2,6 +2,9 @@
 
 module IvcChampva
   class VHA1010d
+    ADDITIONAL_PDF_KEY = 'applicants'
+    ADDITIONAL_PDF_COUNT = 3
+
```

Comment:
These constants exist so that the functionality can differ between forms. For 10-7959A this might change to '1' and 'claimant'